tasks/airdrop-single-item.ts

```diff
@@ -14,11 +14,11 @@ const MAX_TRANSFER_PER_TX = 120;
 const BATCHTRANSFER_ADDRESSES = new Map();
 BATCHTRANSFER_ADDRESSES.set(
   "43114",
-  "0xe0ad9Fab889C32Bb52C547Bc8deAa98f9df04C87"
+  "0x42Ac9B9CA43b2C36Ac1E78eA3bc71905f55a589a"
 );
 BATCHTRANSFER_ADDRESSES.set(
   "43113",
-  "0x43C2d5387547fb6A7FA3E62aFA431505A01E0fef"
+  "0x497f6d5a33f03cdc26edafea19e5e3b7e5cf76e8"
```

Comment:
with batch transfer that we have in production already i think  https://github.com/traderjoe-xyz/mintpeg/blob/9f03b065568b5abaa0654482a3caea1265d781da/tasks/airdrop-single-item.ts#L98 can be removed?

Reply:
Yeah I guess it's not really necessary anymore but I prefer to have my deployer as clean as possible with no approval hanging around